plugins/scaffolder/src/routes.ts

```diff
@@ -19,64 +19,75 @@ import {
   createSubRouteRef,
 } from '@backstage/core-plugin-api';
 
+/** @public */
```

Comment:
Don't think we should be exporting the `externalRouteRefs`, they don't provide any value.